client/src/visualiser-src/binary-search-tree-visualiser/data-structure/GraphicalBST.ts

```diff
@@ -142,6 +144,11 @@ class GraphicalBST extends GraphicalDataStructure {
     return animationProducer;
   }
 
+  public generate(): void {
+      const num = insertBalancedBSTNumbers();
+      num.forEach(element => {this.insert(element)});
```

Comment:
I'm not sure if calling insert/append functions is the best way of going about this because these functions activate the animation timeline, whereas we just want the nodes to appear (without any highlighting or curr pointers moving). You might be better off writing a 'randomlyGenerate' method for each data structure that animates all the nodes into existence without placing the animations on the timeline.